app/controllers/clearance/sessions_controller.rb

```diff
@@ -21,7 +21,11 @@ def destroy
   end
 
   def new
-    render template: 'sessions/new'
+    if current_user
```

Comment:
Can we do this as a before_filter instead? That will make it much simpler to override as you'll just be able to `skip_before_filter :redirect_signed_in_users` or whatever its called.